.github/workflows/ci.yml

```diff
@@ -32,6 +32,30 @@ jobs:
           paths_ignore: '["**.md", "**.png", "**.jpg"]'
           do_not_skip: '["workflow_dispatch", "schedule", "push"]'
 
+  report-breaking-changes:
+    runs-on: ubuntu-22.04
+    needs: detect-noop
+    if: needs.detect-noop.outputs.noop != 'true'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Get modified CRDs
+        id: modified-crds
+        uses: tj-actions/changed-files@v34
+        with:
+          files: |
+            package/crds/**
+
+      - name: Report breaking CRD OpenAPI v3 schema changes
+        if: steps.modified-crds.outputs.any_changed == 'true'
+        env:
+          MODIFIED_CRD_LIST: ${{ steps.modified-crds.outputs.all_changed_files }}
+        run: |
+          mkdir -p .cache/tools/linux_x86_64
```

Comment:
I believe this wouldn't be needed if we bump build submodule to latest including this fix: https://github.com/upbound/build/pull/220/files

Reply:
Done. Validated the new definition with this run:
https://github.com/upbound/provider-aws/actions/runs/3603138332/jobs/6070964961